### PR TITLE
CLN: rename private functions used across modules

### PR DESCRIPTION
**Pull request by jbrockmendel**

### CODE REVIEW

File: pandas/tests/plotting/common.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 from pandas import DataFrame, Series
 import pandas._testing as tm
 
+
 """
 This is a common base class used for various plotting tests
 """
```
Comment (simonjayhawkins) [inline pandas/tests/plotting/common.py:19]: does this docstring belong inside the following class

**Review by simonjayhawkins — APPROVED** [review]
Thanks @jbrockmendel lgtm pending green

File: pandas/tests/plotting/common.py (hunk 1 above)
Comment (jbrockmendel) [reply pandas/tests/plotting/common.py:19]: yes, updated

File: pandas/tests/plotting/test_misc.py [hunk 2]
```diff
@@ -349,31 +349,31 @@ def test_subplot_titles(self, iris):
         title_list = [ax.get_title() for sublist in plot for ax in sublist]
         assert title_list == title[:3] + [""]
 
-    def test_get_standard_colors_random_seed(self):
+    def testget_standard_colors_random_seed(self):
```
Comment (jreback) [inline pandas/tests/plotting/test_misc.py:352]: test_

File: pandas/tests/plotting/test_misc.py [hunk 3]
```diff
@@ -349,31 +349,31 @@ def test_subplot_titles(self, iris):
...
 
-        color1 = _get_standard_colors(1, color_type="random")
-        color2 = _get_standard_colors(1, color_type="random")
+        color1 = get_standard_colors(1, color_type="random")
+        color2 = get_standard_colors(1, color_type="random")
         assert color1 == color2
 
-    def test_get_standard_colors_default_num_colors(self):
-        from pandas.plotting._matplotlib.style import _get_standard_colors
+    def testget_standard_colors_default_num_colors(self):
```
Comment (jreback) [inline pandas/tests/plotting/test_misc.py:370]: test_

File: pandas/tests/plotting/test_misc.py [hunk 4]
```diff
@@ -394,17 +394,17 @@ def test_plot_single_color(self):
         colors = [rect.get_facecolor() for rect in ax.get_children()[0:3]]
         assert all(color == colors[0] for color in colors)
 
-    def test_get_standard_colors_no_appending(self):
+    def testget_standard_colors_no_appending(self):
```
Comment (jreback) [inline pandas/tests/plotting/test_misc.py:397]: test_

**Review by jreback — CHANGES_REQUESTED** [review]
typos, merge on green.

**Review by jreback — APPROVED** [review]

**Pull request merged**